src/electron/platform/android/rule-information-provider.ts

```diff
@@ -92,8 +92,8 @@ export class RuleInformationProvider {
         );
     };
 
-    private buildUnifiedResolution(unformattedText: string, codeStrings: string[] = null): UnifiedResolution {
-        return { howToFixSummary: unformattedText, formatAsCode: codeStrings };
+    private buildUnifiedResolution(unformattedText: string, codeStrings: string[] = null): UnifiedFormatAsCodeResolution {
+        return { howToFixSummary: unformattedText, howToFixFormat: { howToFix: unformattedText, formatAsCode: codeStrings } };
```

Comment:
Why are we duplicating the unformattedText in the object? Lisa and I intentionally changed this recently to avoid exactly this behavior, and I'd be very sad to see it return. My guess is that there's downstream code that consumes the howToFixFormat property. I'll argue that the downstream code should consume the UnifiedFormattedResolution (or whatever we end up calling it)--that way it has all of the information that it needs in a single object, with no duplication.

Reply:
As we discussed offline, we should have a design conversation to reconcile the differences between issue filing (which operates on the entire `UnifiedResolution` object, and cards, which operates on the `howToFixFormat` object. That's outside the scope of this PR, though, so we'll live with temporary duplication of data